Jupyterhub Docker not able to find any jinja templates

I wanted to run jupyterhub on a docker and used the below -
docker run -p 8000:8000 -d --name jupyterhub jupyterhub/jupyterhub jupyterhub
Unable to find the login/error jinja templates

Tried the below code as well still the same -

FROM jupyterhub/jupyterhub:master



RUN apt-get update --fix-missing && apt-get -y install \
    wget git rsync openssh-server build-essential gcc openjdk-8-jdk-headless && \
    apt install -y vim vim-tiny && \
    rm -rf /var/lib/apt/lists/*

COPY requirements.txt /tmp/requirements.txt

RUN pip install -r /tmp/requirements.txt

RUN echo "127.0.0.1 $(hostname) localhost" > /etc/hosts
RUN echo "echo \"127.0.0.1 $(hostname) localhost\" > /etc/hosts" >> /etc/profile

CMD jupyterhub --ip 0.0.0.0 --port 8000 --Spawner.default_url='/lab'

Gives the following error 
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/3715878/70894088-540f2400-2012-11ea-94c9-5f6b27485b3c.png">

I built the image locally, found the following errors in the build log.

```
  Building wheel for jupyterhub (PEP 517): started
  Running command /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/pip/_vendor/pep517/_in_process.py build_wheel /tmp/
tmpr88qla89
  running bdist_wheel
  running build
  running build_py
  running js

  > jupyterhub-deps@0.0.0 postinstall /tmp/pip-req-build-mnx4w6t5
  > python ./bower-lite

  python: can't open file './bower-lite': [Errno 2] No such file or directory

  npm WARN jupyterhub-deps@0.0.0 license should be a valid SPDX license expression
  npm ERR! Linux 3.10.0-1062.1.1.el7.x86_64
  npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "--progress=false" "--unsafe-perm"
  npm ERR! node v8.10.0
  npm ERR! npm  v3.5.2
  npm ERR! code ELIFECYCLE
  npm ERR! jupyterhub-deps@0.0.0 postinstall: `python ./bower-lite`
  npm ERR! Exit status 2
  npm ERR!
  npm ERR! Failed at the jupyterhub-deps@0.0.0 postinstall script 'python ./bower-lite'.
  npm ERR! Make sure you have the latest version of node.js and npm installed.
  npm ERR! If you do, this is most likely a problem with the jupyterhub-deps package,
  npm ERR! not with npm itself.
  npm ERR! Tell the author that this fails on your system:
  npm ERR!     python ./bower-lite
  npm ERR! You can get information on how to open an issue for this project with:
  npm ERR!     npm bugs jupyterhub-deps
  npm ERR! Or if that isn't available, you can get their info via:
  npm ERR!     npm owner ls jupyterhub-deps
  npm ERR! There is likely additional logging output above.

  npm ERR! Please include the following file with any support request:
  npm ERR!     /tmp/pip-req-build-mnx4w6t5/npm-debug.log
```
It seems that `bower-lite` is not found in `/src/jupyterhub` while building wheel.
The `package.json` depends `bower-lite`.

If I add `bower-lite` to the COPY src arguments at https://github.com/jupyterhub/jupyterhub/blob/master/Dockerfile#L45-L50 and rebuild, I get a login page as expected.

--- a/Dockerfile
+++ b/Dockerfile
@@ -41,14 +41,10 @@ RUN apt-get update \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
-# copy only what we need to avoid unnecessary rebuilds
-COPY bower-lite \
-     package.json \
-     pyproject.toml \
-     README.md \
-     requirements.txt \
-     setup.py \
-     /src/jupyterhub/
+# copy everything except whats in .dockerignore, its a
+# compromise between needing to rebuild and maintaining
+# what needs to be part of the build
+COPY . /src/jupyterhub/
 COPY jupyterhub/ /src/jupyterhub/jupyterhub
 COPY share/ /src/jupyterhub/share
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -42,7 +42,8 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/*
 
 # copy only what we need to avoid unnecessary rebuilds
-COPY package.json \
+COPY bower-lite \
+     package.json \
      pyproject.toml \
      README.md \
      requirements.txt \

